packages/meditrak-server/src/routes/importSurveyResponses/importSurveyResponses.js

```diff
@@ -39,7 +39,12 @@ export async function importSurveyResponses(req, res) {
       throw new UploadError();
     }
     const surveyNames = getArrayQueryParameter(req?.query?.surveyNames);
-    const timeZone = req?.query?.timeZone || DEFAULT_DATABASE_TIMEZONE;
+    const timeZone = req?.query?.timeZone;
```

Comment:
The change here is simpler, we just require the timezone which then gets stripped. Since it's a 1:1 mapping of endpoint and client, we don't need the endpoint to be as generic as postChanges. We should clean this up in the future perhaps.